### PR TITLE
Buiding layouts and view

### PR DESCRIPTION
**Pull request by xuannam79**

### CODE REVIEW

**Comment by xuannam79** [comment]
ready

File: resources/lang/en/messages.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+return [
+    'task' => 'Task',
+    'addTask' => 'Add Task',
+];
```
Comment (tranducquoc) [inline resources/lang/en/messages.php]: thêm end file

File: resources/lang/vi/messages.php [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?php
+return [
+    'task' => 'Công việc',
+    'addTask' => 'Thêm công việc',
+];
```
Comment (tranducquoc) [inline resources/lang/vi/messages.php]: thêm end file

File: resources/views/common/errors.blade.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
+@if (count($errors) > 0)
+    <div class="alert alert-danger">
+        <strong>Whoops! Something went wrong!</strong>
```
Comment (tranducquoc) [inline resources/views/common/errors.blade.php]: i18n

File: resources/views/common/errors.blade.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+        <br><br>
+
+        <ul>
+            @foreach ($errors->all() as $error)
+                <li>{{ $error }}</li>
+            @endforeach
+        </ul>
+    </div>
+@endif
```
Comment (tranducquoc) [inline resources/views/common/errors.blade.php]: thêm end file

File: resources/views/layouts/app.blade.php [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    <body>
+        <div class="container">
+            <nav class="navbar navbar-default">
+
+            </nav>
+        </div>
+
+        @yield('content')
+    </body>
+</html>
```
Comment (tranducquoc) [inline resources/views/layouts/app.blade.php]: thêm end file

File: resources/views/tasks/index.blade.php [hunk 6]
```diff
@@ -0,0 +1,25 @@
+@extends('layouts.app')
+
+@section('content')
+
+
```
Comment (tranducquoc) [inline resources/views/tasks/index.blade.php]: remove 1 line

File: resources/views/tasks/index.blade.php [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+        	<div class="form-group">
+        		<div class="col-sm-offset-3 col-sm-6">
+        			{!! Form::button('<i class="fa fa-plus"></i> '.trans('messages.addTask'), ['class'=>'btn btn-default']) !!}
+        		</div>
+        	</div>
+        {!! Form::close() !!}
+
+    </div>
+
+@endsection
```
Comment (tranducquoc) [inline resources/views/tasks/index.blade.php]: thêm end file

**Comment by xuannam79** [comment]
ready

File: resources/views/layouts/app.blade.php [hunk 8]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="{{ app()->getlocale() }}">
+    <head>
+        <title>{{ trans('messages.task') }}</title>
+        <link rel="stylesheet" type="text/css" href="/css/app.css">
```
Comment (tranducquoc) [inline resources/views/layouts/app.blade.php]: dùng laravel colective nhé em

File: resources/views/tasks/index.blade.php [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+        {!! Form::open(['route'=>'tasks.index', 'method'=>'POST', 'class'=>'form-horizontal']) !!}
+        	<div class="form-group">
+        		{!! Form::label('task-name', trans('messages.task'), ['class'=>'col-sm-3 control-label']) !!}
+        		<div class="col-sm-6">
+                    {!! Form::text('name', '', ['id'=>'task-name', 'class'=>'form-control']) !!}
+                </div>
+        	</div>
+        	<div class="form-group">
+        		<div class="col-sm-offset-3 col-sm-6">
+        			{!! Form::button('<i class="fa fa-plus"></i> '.trans('messages.addTask'), ['class'=>'btn btn-default']) !!}
```
Comment (tranducquoc) [inline resources/views/tasks/index.blade.php]: dùng form submit

**Comment by xuannam79** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**